src/transferentropy/bbnue.jl

```diff
@@ -0,0 +1,107 @@
+using TimeseriesSurrogates
+
+export BBNUE
+
+"""
+    BBNUE(est) <: TransferEntropyEstimator
+
+The bootstrap-based non-uniform embedding estimator (BB-NUE) for conditional transfer entropy (Montalto et al., 2014).
+Uses the estimator `est` to compute relevant marginal entropies. (e.g. `VisitationFrequency(RectangularBinning(3))`) 
+
+[^Montalto2014]: Montalto, A.; Faes, L.; Marinazzo, D. MuTE: A MATLAB toolbox to compare established and novel estimators of the multivariate transfer entropy. PLoS ONE 2014, 9, e109462.
+"""
+struct BBNUE{E} <: TransferEntropyEstimator
+    est::E
+end
+
+
+""" 
+    transferentropy(source, target, [cond], est::BBNUE; q = 0.95, η = 1, 
+        nsurr = 100, uq = 0.95, 
+        include_instantaneous = true, 
+        method_delay = "ac_min", 
+        maxlag::Union{Int, Float64} = 0.05
+        ) → te, js, τs, idxs_source, idxs_target, idxs_cond
+
+Estimate transfer entropy using the [`BBNUE`](@ref) estimator, which uses a bootstrap-based
```

Comment:
I vote that all of this docstring is in `BBNUE` instead. "Polluting" the `transferentropy` docstring with so much details about a specific method isn't ideal for a user. The API is clear that `est` is used in `transferentropy`, there is no gain in highlighting here what `transferentropy` does with this specific method. Instead, better to highlight it at `BBNUE`.